tests/src/Mod/Part/App/FeaturePartCut.cpp

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+#include "gtest/gtest.h"
+
+#include "Mod/Part/App/FeaturePartCut.h"
+#include <src/App/InitApplication.h>
+
+#include "PartTestHelpers.h"
+
+class FeaturePartCutTest: public ::testing::Test, public PartTestHelpers::PartTestHelperClass
+{
+protected:
+    static void SetUpTestSuite()
+    {
+        tests::initApplication();
+    }
+
+
+    void SetUp() override
+    {
+        createTestFile();
+        _cut = static_cast<Part::Cut*>(_doc->addObject("Part::Cut"));
+    }
+
+    void TearDown() override
+    {}
+
+
+    Part::Cut* _cut;
+};
+
+TEST_F(FeaturePartCutTest, testIntersecting)
+{
+    // Arrange
+    _cut->Base.setValue(_box1obj);
+    _cut->Tool.setValue(_box2obj);
+
+    // Act
+    _cut->execute();
+    Part::TopoShape ts = _cut->Shape.getValue();
+    double volume = PartTestHelpers::getVolume(ts.getShape());
+    EXPECT_DOUBLE_EQ(volume, 3.0);
```

Comment:
Looks like in this file many tests are missing the //assert. I'm guessing that's a copy paste ;)